Resources/Prototypes/Body/Parts/animal.yml

```
@@ -72,6 +74,7 @@
     - state: l_foot
   - type: BodyPart
     partType: Foot
+    slotId: feet # Shitmed
```

yamllint (1.35.1)

[warning] too few spaces before comment

(comments)